docs/guides/Guide-Nested.md

```diff
@@ -72,3 +72,43 @@ Now we have put one navigator inside another, and we can `navigate` between navi
 ```phone-example
 nested
 ```
+
+## Side Note - Nesting a Navigator in a Component
+Sometimes it is desirable to embed a navigator in a component. This is useful in cases where the navigator only takes up part of the screen. For the child navigator to be wired into the navigation tree, it needs the `navigation` property from the parent navigator.
+
+```js
+const SimpleApp = StackNavigator({
+  Home: { screen: EmbeddingScreen },
```

Comment:
Nit: Change `EmbeddingScreen` to `NestedScreen` or some wording that doesn't use the phrase "embed." This fits better with the common terminology used in the React community to "nest" components.